poly-commitment/benches/ipa.rs

```diff
@@ -8,6 +8,37 @@ use poly_commitment::{
     commitment::CommitmentCurve, ipa::SRS, utils::DensePolynomialOrEvaluations, PolyComm, SRS as _,
 };
 
+fn benchmark_ipa_commit(c: &mut Criterion) {
```

Comment:
Nit: no strong opinion, but it took me a minute or so to find/remember how to run this benchmark only. Would it be worth to add a comment saying how to run this?
```
cargo bench -- "IPA Commit/commitment"
```